js/gratipay/edit_team.js

```diff
@@ -0,0 +1,51 @@
+Gratipay.edit_team = {}
+
+Gratipay.edit_team.initForm = function() {
+    $form = $("#edit-team");
+    $buttons = $form.find("button");
+
+    $form.find(":input").each(function() {
+        // Save original values.
+        $(this).data('original', $(this).val());
+    });
+    $form.submit(Gratipay.edit_team.submitForm);
+}
+
+Gratipay.edit_team.submitForm = function(e) {
+    e.preventDefault();
+
+    var modified = false;
+
+    var data = new FormData();
+    $form.find(":input").each(function() {
+        if($(this).data("original") !== $(this).val()) {
+            modified = true;
+
+            var name = $(this).attr("name");
+            if (name === "image") {
+                data.append(name, $(this)[0].files[0]);
+            } else {
+                data.append(name, $(this).val());
```

Comment:
Why the unnecessary overhead if we are only interested in attributes that changed? Only passing necessary attributes also fits well with the [update](https://github.com/gratipay/gratipay.com/pull/3923/files#diff-aa4aac98a6ec2173f86c51a728870f09R70) method.